### PR TITLE
Remove reactive refs

### PR DESCRIPTION
**Pull request by rnorth**
Having gone through the tutorial from scratch again, I noticed a few things that I missed in #13 - apologies.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id "org.springframework.boot" version "2.1.4.RELEASE"
+    id "org.springframework.boot" version "2.0.2.RELEASE"
```
Comment (rnorth) [inline build.gradle]: Reverting due to changed Spring Kafka startup behaviour, which means that until `KafkaContainer` is added to tests, the service won't start up.

File: step-7-test-the-api.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ public class RatingsControllerTest extends AbstractIntegrationTest {
                 .when()
                 .post("/ratings")
                 .then()
-                .statusLine("HTTP/1.1 400 Bad Request");
+                .statusCode(400);
```
Comment (rnorth) [inline step-7-test-the-api.md]: I'm sure that `statusLine` was working a couple of weeks ago, but either way, the service is not returning a status reason code any more. We're getting back, e.g. `HTTP/1.1 202 ` (note amazingly ugly trailing space!)

[This doesn't seem to be a bug](https://github.com/spring-projects/spring-boot/issues/6789), so let's just make the tests match the actual behaviour.

File: step-8-edge-cases.md [hunk 3]
```diff
@@ -38,20 +38,15 @@ public class RatingsRepositoryTest {
...
-}
+    @Before
+    public void setUp() {
+        LettuceConnectionFactory connectionFactory = new LettuceConnectionFactory(
+                ?,
+                ?
+        );
+        connectionFactory.afterPropertiesSet();
+        repository = new RatingsRepository(new StringRedisTemplate(connectionFactory));
+    }
```
Comment (rnorth) [inline step-8-edge-cases.md]: Missed in #13
